### ISSUE TITLE
Can 'oss-branding' be safely removed?

### ISSUE THREAD

**Issue by kaspersorensen** [datacleaner/DataCleaner#1744]
I wonder what the new-ish module called oss-branding is there for. I see that all the images etc. that are in there are _also_ in the desktop/ui module. Or at least so it seems based on a sampling.

I also tried removing the oss-branding dependency from desktop/ui just to see what happened ... and it was nothing. So I wonder: Can we safely remove oss-branding so that the final distribution of DC does not include these resources twice?

Maybe @losd or @ClaudiaPHI can answer this please?

**Comment by LosD** [comment]
Yes, it can. It was a preliminary effort to be able to separate OSS and commercial icons.

Though I'd recommend instead embracing and using it. The current way is beyond horrible.

**Comment by kaspersorensen** [comment]
Ok but then you would delete the copies in desktop/ui right?

**Comment by kaspersorensen** [comment]
Hmm I was looking into this and I think the filenames/paths of `oss-branding` doesn't match so that it could be a drop-in replacement for the images in `desktop/ui`. For instance, in `IconUtils` we have paths like `images/model/dictionary.png` which matches a file in  `desktop/ui`. But in `oss-branding` the corresponding file is under `model/dictionary` (without the `images/` part). So that makes it a very tedious task to actually replace this "safely", doesn't it?

**Comment by LosD** [comment]
I'm not sure about very tedious, but there's certainly work to do.
